src/schematools/cli.py

```diff
@@ -375,8 +388,9 @@ def validate(
     \b
         DATASET_ID: id of the dataset.
         META_SCHEMA_URL: URL where the meta schema for Amsterdam Schema definitions can be found.
+        If multiple are given, schematools will try to validate against the largest version,
+        working backwards and stopping at the first version that the objects are valid against.
```

Comment:
This is rather complicated. What is the use case? Can't it be handled by just calling `schema validate` twice?